### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by nizam-betterapp** [mayankmkh/linear-app-plugin#205]
During the setup process, "Test" button says connection successful. But when I see the tasks, it says "Couldn't connect" with Configure server option.

**Comment by nizam-betterapp** [comment]
![Uploading Screenshot 2024-01-31 at 10.42.34 AM.png…]()

**Comment by nizam-betterapp** [comment]
<img width="642" alt="Screenshot 2024-01-31 at 10 42 34 AM" src="https://github.com/mayankmkh/linear-app-plugin/assets/128673271/b6d2b26f-ffb7-49ad-99fa-d30220093514">
<br/>
<br/>

<img width="404" alt="Screenshot 2024-01-31 at 10 42 50 AM" src="https://github.com/mayankmkh/linear-app-plugin/assets/128673271/7ed50adc-2802-4dd6-9131-966b45dd0ff3">

**Comment by dennisholmer** [comment]
I get this too

**Comment by igordrnobrega** [comment]
Any updates on this?

**Comment by huyz** [comment]
While you wait, you can build your own copy using my PR https://github.com/mayankmkh/linear-app-plugin/pull/211

**Comment by Moinax** [comment]
It's been a while now that a PR has been merged for this, but I don't see any new version coming. 

Is there a release plan, because I'm used to use this all the time with Jira and really wish I can switch to Linear.

Thanks in advance

**Comment by SirMishaa** [comment]
@mayankmkh Sorry for the ping

Since the PR is merged, could u please prepare a release ? This is a good plugin, and I'd like to avoid forking it to make it works

**Comment by huyz** [comment]
@SirMishaa are you sure the PR is merged?

**Comment by SirMishaa** [comment]
> @SirMishaa are you sure the PR is merged?

https://github.com/Benbentwo/linear-app-plugin/pull/1

**Comment by huyz** [comment]
@SirMishaa https://github.com/Benbentwo/linear-app-plugin/ is a different repo, forked from https://github.com/mayankmkh/linear-app-plugin/

**Comment by Benbentwo** [comment]
Correct, That link is my personal fork @SirMishaa. Feel free to use it, though I have no plans, nor time ATM, to maintain this. You can use my fork to mess around with the plugin, but ideally the above PR would be merged.

Sorry for any confusion on merges, I prefer to link related issues.
